internal/command/status/machines.go

```diff
@@ -9,10 +9,41 @@ import (
 	"github.com/superfly/flyctl/api"
 	"github.com/superfly/flyctl/client"
 	"github.com/superfly/flyctl/flaps"
+	"github.com/superfly/flyctl/internal/logger"
 	"github.com/superfly/flyctl/internal/render"
 	"github.com/superfly/flyctl/iostreams"
 )
 
+func getFromMetadata(m *api.Machine, key string) string {
+	if m.Config != nil && m.Config.Metadata != nil {
+		return m.Config.Metadata[key]
+	}
+
+	return ""
+}
+
+func getProcessgroup(m *api.Machine) string {
+	name := getFromMetadata(m, api.MachineConfigMetadataKeyFlyProcessGroup)
+
+	if name != "" {
+		return name
+	}
+
+	return "<default>"
+}
+
+func getReleaseVersion(m *api.Machine) string {
+	return getFromMetadata(m, api.MachineConfigMetadataKeyFlyReleaseVersion)
+}
+
+func getImage(m []*api.Machine) string {
```

Comment:
there will be situations where there are different versions of a release deployed to different machines. this will happen when flyctl gets interrupted.

we could remove the image or maybe try to pick the one associated with the highest numbered release?